sktime/forecasting/ttm.py

```diff
@@ -35,6 +35,28 @@ class TinyTimeMixerForecaster(_BaseGlobalForecaster):
     open-sourced by IBM Research. With less than 1 Million parameters, TTM introduces
     the notion of the first-ever "tiny" pre-trained models for Time-Series Forecasting.
 
+    **Zero-shot and Fine-tuning**:
+    The Model serves as a foundation model capable of both zero-shot forecasting and
+    fine-tuning on custom datasets. The decision to fine-tune is handled internally and
+    is determined by the configuration specified during model loading.
+
+    - **Zero-Shot Forecasting**: When the default configuration remains unchanged,
```

Comment:
Are you sure that this is the case. I think it is only fitted if the config in the end led to changes in the size of the weights. I.e., if dimensions of at least on weight layer changes or if they are renamed (if condition with the mismatched keys).